### DISCUSSION TITLE
Impfzertifikatsserver offline

### DISCUSSION THREAD

**Discussion by msteffenDev** [Digitaler-Impfnachweis/certification-apis#163] [Q&A]
Gibt es Probleme in der Produktivumgebung?
Bei uns gibt es viele Rückmeldungen, dass die Impfzertifikate nicht abgerufen werden können,
gestern hat es noch funktioniert.

**Comment by AZwickler** [comment]
> Gibt es Probleme in der Produktivumgebung?
> Bei uns gibt es viele Rückmeldungen, dass die Impfzertifikate nicht abgerufen werden können,
> gestern hat es noch funktioniert.

Hallo, wir haben das gleiche Problem.

**Reply by mge-medisoftware** [reply]
Wir auch.
In den letzten 8 stunden konnte kein einziges Impfzertifikat erfolgreich erstelt werden, Fehler: Unauthorized (ERR_HTTP_401) in Phase: Authentifikation: überermittle signierte challenge und öffentliches Zertifikat SMCB

**Reply by mge-medisoftware** [reply]
Hmm, um 9:36 wurde scheinbar was geändert bei Impfserver, nun anderer Fehler:
Bad Request (ERR_HTTP_400) in Phase: Authentifikation: überermittle signierte challenge und öffentliches Zertifikat SMC-B

**Comment by marcusthierfelder** [comment]
Hat es evtl damit zu tun?

https://fachportal.gematik.de/ti-status#c4381

> 
letzte Aktualisierung: 14.07.2021 18:30 Uhr

Seit dem 14.07.2021 kann es bei einigen Betriebskrankenkassen, bei Innungskrankenkassen und bei der DAK beim Abgleich der Versichertenstammdaten zu Problemen kommen. Als Ursache wurde ein abgelaufenes OCSP-Signer-Zertifikat des bereitstellenden Trusted Service Providers identifiziert. Durch ein ad-hoc initiiertes TSL-Update, welches bereits am Downloadpunkt veröffentlicht wurde, wird wieder ein valides Zertifikat bereitgestellt.

Bei betroffenen Leistungserbringern wird das benannte Problem durch eine Aktualisierung der TSL behoben.

**Reply by thinkberg** [reply]
Wir schauen das Problem gerade an. Als erstes werden wir das TSL Update einspielen.

**Reply by marcusthierfelder** [reply]
vielen dank

**Reply by oh-t2med** [reply]
Vielen Dank!

**Reply by marcusthierfelder** [reply]
bis vorhin kam "Certificate & Signature Verification failed"
jetzt kommt "Unexpected error during auth flow"

**Reply by thinkberg** [reply]
Das TSL file ist aktualisert.

**Reply by mge-medisoftware** [reply]
Vielen Dank! Aber funktioniert leider noch nicht, nun HTTP-Fehler 400 in Phase: Authentifikation: überermittle signierte challenge und öffentliches Zertifikat SMC-B

**Reply by arnoldmaik** [reply]
Ja hier leider auch noch kein positives feedback. Fehler besteht offenbar weiterhin.

**Reply by djalexz** [reply]
Hallo zusammen,

wir haben die TLS ebenfalls aktualisiert und nun erscheint diese Fehlermeldung:
![image](https://user-images.githubusercontent.com/11389551/125753610-13c912d3-fdff-4b1e-b307-54ac7b337c85.png)

Evtl. hilft das ja weiter.

VG Alex

**Reply by arnoldmaik** [reply]
Wir konnten gerade in mehreren Praxen wieder Impfzertifikate erstellen, kann das jemand bestätigen. Kann die Störung damit als behoben betrachtet werden?

**Reply by oh-t2med** [reply]
Bei uns konnten auch Anwender wieder Zertifikate erstellen. Das sieht also gut aus.

**Reply by marcusthierfelder** [reply]
vielen dank, geht wieder

**Reply by djalexz** [reply]
Jawohl, alles wieder gut.
Vielen Dank

**Reply by mge-medisoftware** [reply]
Ja, vielen dank. Seit 12:05 läufts wieder.

**Reply by thinkberg** [reply]
Super. Sorry fuer die Probleme. Es ist nicht so einfach fuer uns PU TI issues zu identifizieren. Die TSL file Aktualisierung war relativ schnell durch, aber leider ist dabei ein defektes CA file (zur Verifizerung der TSL Zertifikate) ausgerollt worden. Dies zu identifizieren hat leider etwas gebraucht. Entsprechende Massnahmen um a) Updates des TSL files rechtzeitig zu erkennen und auszurollen und b) derartige Defekte zu erkennen vor dem Deployment werden jetzt vorgenommen.
